src/qt/listbox.cpp

```diff
@@ -147,14 +147,15 @@ wxString wxListBox::GetString(unsigned int n) const
     return wxQtConvertString( item->text() );
 }
 
-void wxListBox::SetString(unsigned int n, const wxString& WXUNUSED(s))
+void wxListBox::SetString(unsigned int n, const wxString& s)
 {
     QListWidgetItem* item = m_qtListWidget->item(n);
     wxCHECK_RET(item != NULL, wxT("wrong listbox index") );
     if ( m_hasCheckBoxes )
     {
         item->setFlags(Qt::ItemIsUserCheckable | Qt::ItemIsEnabled | Qt::ItemIsSelectable);
         item->setCheckState(Qt::Unchecked);
+        item->setText(QString(s));
```

Comment:
How does this work?
There's a helper function for converting between `wxString` and `QString` AFAIK.

Reply:
You are absolutely correct, I have updated the change to utilise: `wxQtConvertString` rather than a straight up type-cast. Thanks!